Translation of hello.rst

pawel-lewtak opened the pull request:
Translated to Polish first page of the tutorial.

pawel-lewtak committed:
First translation for hello.rst

pawel-lewtak committed:
Few fixes for hello.rst

pawel-lewtak committed:
Final fixes

WojtekReu committed:
Merge pull request #1 from plpug/translate_hello_world

Translate hello.rst

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+...mówi MicroPythonowi, by zaimportował wszystkie rzeczy potrzebne do pracy
+z BBC micro:bit. Wszystko to jest w module ``microbit`` (moduł to biblioteka
+z wcześniej przygotowanym kodem). Poleceniem ``import`` mówisz MicroPythonowi,
+że chcesz użyć danego modułu, a ``*`` to sposób Pythona, na określenie *wszystkiego*.
```
niepotrzebny przecinek

pawel-lewtak replied:
Poprawione.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+z wcześniej przygotowanym kodem). Poleceniem ``import`` mówisz MicroPythonowi,
+że chcesz użyć danego modułu, a ``*`` to sposób Pythona, na określenie *wszystkiego*.
+Zatem ``from microbit import *`` oznacza "chcę użyć wszystkich dostępnych funkcji
+z biblioteki microbit".
```
Nie tylko funkcji, także zmiennych, klas, stałych, obiektów etc. W oryginalne jest "wszystkiego" a nie "wszystkich funkcji".

pawel-lewtak replied:
Trochę to zmieniłem, żeby lepiej oddawało sens.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+world!". ``display`` w tym przypadku to *obiekt*
+(ang. object) z modułu ``microbit``, który reprezentuje fizyczny wyświetlacz
+urządzenia (mówimy "obiekt" zamiast "rzecz" lub "to coś").
+By wydać wyświetlaczowi polecenie, używamy jego nazwy po kropce ``.`` - tak
```
Niejasne do czego tutaj odnosi się "jego". Może coś w rodzaju "wpisujemy po kropce komendę"?

deshipu replied:
Minus "-" zamiast myślnika "--".

pawel-lewtak replied:
Poprawione.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+używamy polecenia ``scroll`` (ang. przewiń). Polecenie ``scroll``
+musi wiedzieć jakie znaki pokazać na wyświetlaczu. Wpisujemy je ujęte
+w cudzysłów (``"``) i zamknięte w nawiasy (``(`` i ``)``). W programowaniu nazywamy
+to *argumentami* (ang. arguments). Zatem ``display.scroll("Hello, world!")``
```
Ponownie niejasne, do czego odnosi się tutaj "to".

pawel-lewtak replied:
Poprawione.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+musi wiedzieć jakie znaki pokazać na wyświetlaczu. Wpisujemy je ujęte
+w cudzysłów (``"``) i zamknięte w nawiasy (``(`` i ``)``). W programowaniu nazywamy
+to *argumentami* (ang. arguments). Zatem ``display.scroll("Hello, world!")``
+oznacza, po angielsku, "chcę użyć wyświetlacza by pokazać przesuwający się
```
po polsku

pawel-lewtak replied:
Poprawione.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+tekst 'Hello, world!'". Jeśli metoda nie potrzebuje żadnych argumentów,
+musimy to jasno określić używając pustych nawiasów: ``()``.
+
+Skopiuj powyższy kod do swojego edytora i zaprogramuj go (ang. flash) na
```
proponuję użyć słowa "wgraj", po "zaprogramuj" jest trochę zbyt wieloznaczne

pawel-lewtak replied:
"Wgraj" wydaje się trochę niszowe, może nie być zrozumiałe dla kogoś kto nie programował wcześniej. Ale nie upieram się, mogę zmienić jeśli chcesz.

deshipu replied:
Wydaje mi się, że jeśli już uczymy programować, to powinniśmy też przy okazji uczyć precyzyjnej terminologii, żeby potem przy kontynuowaniu nauki nie trzeba było wszystkiego "oduczać" i uczyć od nowa z poprawnym słownictwem. Słowo "flash" w języku angielskim także jest raczej niszowe, normalnie znaczy zupełnie co innego.

pawel-lewtak replied:
Słuszna uwaga. Zaraz poprawię.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+    między "Pawłem" i "Gawłem" - dwa różne imiona, choć brzmią i wyglądają
+    podobnie.
+    
+    Natomiast jeśli w treści błędu zobaczysz ``SyntaxError``, oznacza to po
```
Czy warto wspomnieć, że to znaczy "błąd składni"?

pawel-lewtak replied:
Dodane.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+
+    Python oczekuje, że wprowadzisz **BEZBŁĘDNY** kod. Na przykład, ``Microbit``,
+    ``microbit`` i ``microBit`` są przez Pythona traktowane jak trzy osobne
+    rzeczy. Jeśli w treści błędu zobaczysz ``NameError``, oznacza to że
```
być może warto dodać w nawiasie tłumaczenie nazwy błędu

pawel-lewtak replied:
Dodane.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+    gdyby umieścić. kropkę w środku zdania. Trudno jest wtedy zrozumieć co 
+    autor miał na myśli.
+    
+    Twój microbit może przestać odpowiadać: nie uda się zaprogramować nowego
```
reagować

pawel-lewtak replied:
Poprawione.

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,75 @@
+    kodu lub wpisywać poleceń w konsoli REPL. W takim przypadku spróbuj
+    odłączyć urządzenie od prądu i podłączyć ponownie po krótkiej chwili.
+    Chodzi o to, by odłączyć kabel USB (i kabel baterii jeśli jest podłączony),
+    a po chwili podłączyć ponownie. Najpewniej będzie należało również
```
Może być także konieczne...

pawel-lewtak replied:
Poprawione.

deshipu reviewed (changes requested):
Bardzo dziękuję za pierwsze tłumaczenie.

Będę okrutny i będę się strasznie czepiał wszystkiego, z dokładnością do każdego przecinka. Z doświadczenia wiem, że łatwiej jest takie rzeczy poprawiać od razu, niż po włączeniu zmian do repozytorium. Chcę przy tym zaznaczyć, że moje faszystowskie podejście nie oznacza, że uważam, że autor się nie zna, albo że ja próbuję się popisywać. Błędy popełnia każdy, a po to jest druga para oczu, żeby je wyłapywać i sugerować ulepszenia, więc mam nadzieję, że nie będziecie mi tego mieć za złe.

Swoją drogą, wygląda to lepiej niż moje pierwsze podejścia.

pawel-lewtak commented:
Dziękuję za uwagi, dodatkowa para oczu do wyłapania błędów jest na wagę złota :-)
Poprawię wymienione rzeczy.

pawel-lewtak committed:
Update hello.rst according to review comments

Fixed as suggested by deshipu in https://github.com/deshipu/microbit-micropython-pl/pull/3

pawel-lewtak committed:
Change translation of word "flash"

deshipu commented on docs/tutorials/hello.rst:
```diff
@@ -1,72 +1,76 @@
+nie potrzebuje żadnych argumentów, musimy to jasno określić używając pustych 
+nawiasów: ``()``.
+
+Skopiuj powyższy kod do swojego edytora i wgraj go (ang. flash) na urządzeniu.
```
Wgraj go do urządzenia?

pawel-lewtak replied:
Za szybko chciałem poprawić, będę uważał na przyszłość.

deshipu reviewed (commented):
Został drobny błąd gramatyczny.

pawel-lewtak committed:
Grammar fix

deshipu commented:
Wydaje mi się, że teraz jest wszystko dobrze. Bardzo dziękuję za tłumaczenie!

pawel-lewtak commented:
Dziękuję za uwagi :-)
Kolejne tłumaczenia mam nadzieję wkrótce.
